src/Resources/app/storefront/src/scss/category-filter.scss

```diff
@@ -11,6 +11,11 @@
   .custom-checkbox {
     max-width: 290px;
 
+    &.disabled,
+    &:disabled {
+      opacity: calc(.3/.5);
```

Comment:
This causes some really odd opacity, until the point comes where it is actually almost invisible:
![image](https://user-images.githubusercontent.com/20555112/108477725-e30d7100-7293-11eb-8535-6c0240b79f45.png)

Please remove this and let it handle like before.

Reply:
Disable class is applied differently for the category filter due to nested values. Which is why it adds opacity for deeper nested links. If you want to just disable the filter, that I can do and it will use the default styling instead of using opacity. But then the disable styling for category will be different than other filters.